Magento/ruleset.xml

```diff
@@ -305,6 +306,7 @@
     <rule ref="PSR2.Classes.PropertyDeclaration">
         <severity>6</severity>
         <type>warning</type>
+        <exclude name="PSR2.Classes.PropertyDeclaration.Underscore"/>
```

Comment:
Why are these excluded? We do not want any variables and method with underscore prefixes.

Reply:
It's because a lot of legacy code in Magento. When extension developers extend some classes they need to suppress such findings.

Reply:
We need some strategy for the new code to enable such rules there.